application/server/server.go

```diff
@@ -638,13 +638,19 @@ func textDocumentDidOpenHandler() jrpc2.Handler {
 		logger := c.Logger().With().Str("method", "TextDocumentDidOpenHandler").Str("documentURI", filePath).Logger()
 
 		logger.Info().Msg("Receiving")
-		folder := workspace.Get().GetFolderContaining(filePath)
+		folder := c.Workspace().GetFolderContaining(filePath)
```

Comment:
let's pass the config to this eventHandler 
maybe we can get rid of all config.CurrentConfig all together here

Reply:
ok